src/secrets.js

```diff
@@ -35,7 +35,12 @@ async function getSecrets(secretRequests, client) {
             cachedResponse = true;
         } else {
             try {
-                const result = await client.get(requestPath);
+                let result;
+                if (bodyReq) {
```

Comment:
The variable is currently unresolved, did you mean to name the new `body` argument passed to getSecrets `bodyReq` instead?